weblate/addons/tests.py

```diff
@@ -678,6 +682,30 @@ def test_customize_tabs(self) -> None:
         )
         self.asset_customize("\t\t\t\t\t\t\t\t")
 
+    def test_customize_compact_mode_on(self) -> None:
+        JSONCustomizeAddon.create(
+            component=self.component,
+            configuration={
+                "indent": 4,
+                "sort": 1,
```

Comment:
Shouldn't this be sort_keys instead of sort? I used sort because that's what some other tests in the same class use.